src/seml/utils/__init__.py

```diff
@@ -430,9 +431,12 @@ def __init__(
         name: str,
```

Comment:
why again do we have to cache function results to the disc? Why not use `functors.lru_cache`?

Reply:
Ok, I guess it is to cache collections and the cache needs to persist in between python contexts of seml CLI calls...